spring-cloud-contract-stub-runner/src/test/groovy/org/springframework/cloud/contract/stubrunner/ContractDownloaderSpec.groovy

```diff
@@ -32,36 +30,31 @@ class ContractDownloaderSpec extends Specification {
 
 	def 'should set inclusion pattern on config when path pattern was explicitly provided with a separator at the beginning'() {
 		given:
-			String contractPath = File.separator + ['a', 'b', 'c', 'd'].join(File.separator)
-			ContractDownloader contractDownloader = new ContractDownloader(stubDownloader,
+			def contractPath = File.separator + ['a', 'b', 'c', 'd'].join(File.separator)
```

Comment:
I definitely prefer to have those typed :)

Reply:
@marcingrzejszczak I saw multiple styles of using this within the project: sometimes it's `def`, sometimes exact type. `def` is quite concise, so I used it here. What is the rule in general? I'm not so familiar with groovy, but I thought that type will actually be derived based on the assigned value (i.e. similar to `var` in the latest java, except for that you can reassign it). That seems to be not so bad... Do you prefer to define concrete types everywhere instead?

Reply:
Yeah, so I prefer those to be explicit always. I know that they are not done in such a way everywhere, but I try to make those explicit. Groovy is a dynamic language so sometimes the compiler gets lost on what the type is.